js/modules/login.js

```diff
@@ -1,15 +1,9 @@
-export const login = document.getElementById('login')
-export const modal = document.getElementById('modalContainer')
-
-export const appearBtn = () => { modal.style.display = 'block' }
-export const hideBtn = () => { modal.style.display = 'none' }
-
-window.onclick = (event) => {
-    if (event.target == modal) {
-        modal.style.display = 'none';
-    }
+export const appearById = (element) => {
+    const appearId = document.getElementById(element)
+    appearId.style.display = 'block'
```

Comment:
think about the correctness of the naming... You should be receiving an id in the function, since you are using `getElementById`, so you should be sending an id (normally of type string)

also what you get is an `element`, normally an object, so please update the naming of both variables, they are the other way around